File: 02/ans.rs
#[path = "../util.rs"]
mod util;
use util::read_input;

fn main() {
    let lines = read_input("sample.txt");
    let (a, b) = solution(lines);
    println!("{0}, {1}", a, b);
}


fn solution(lines: Vec<String>) -> (i32, i32) {
    (0, 1)
}

File: ans.py
import sys
from pathlib import Path
sys.path.append(str(Path(__file__).parent.parent))
from helpers import read_input

def solve(data):
    """Solution method"""
    return 0

def main():
    # Read input
    input_data = read_input(str(Path(__file__).parent / 'input.txt'))
    
    # Solve the problem
    answer = solve(input_data)
    
    # Print result
    print(f"Answer: {answer}")

if __name__ == "__main__":
    main()